geoutils/georaster.py

```diff
@@ -1440,20 +1444,43 @@ def interp_points(self,pts,nsize=1,mode='linear',band=1):
 
         return rpts
 
-    def split_bands(self):
+    def split_bands(self, copy=False, subset: Optional[Union[list[int], int]] = None):
```

Comment:
What is the idea behing the Optional class? Isn't it obvious by the fact that it has a default value?
Otherwise, why is it not needed for copy, for example?

Reply:
Ok I better understand now! Type hinting is tough!! Good that we have a pro among us ;-)